sherpa/optmethods/ncoresde.py

```diff
@@ -531,12 +531,12 @@ def __call__(self,
                 if tmp.statval < best_fval:
                     best_par = np.append(tmp.pars, tmp.statval)
                     mypop[1] = best_par[:]
-                    self.polytope.sort()
+                    mypop.sort()
                     old_fval = tmp.statval
                 else:
                     old_fval = best_fval
 
-        best_vertex = self.polytope[0]
+        best_vertex = mypop[0]
```

Comment:
If `self.polytope` is exactly the same as `mypop` then why do we need the symbol `mypop` at all? Would it not be clearer to use use one and only one name for the thing?

Reply:
So, this is old code and I am just trying to make it more consistent. We can either have `mypop` or `self.polytope` and `mypop` has the advantage of being shorter (if you look at the code before I started it had a mix of `mypop` and `self.polytope` in). The advantage of using `self.polytope` is that you are reminded each time that you are dealing with an element in `self`. However, I have sometimes written code that will label an attribute of `self` just to make it shorter/easier to read.

Reply:
I've updated the code to just use `self.polytope`.